kubernetes/setup-environment

```diff
@@ -11,8 +11,16 @@ done
 
 log "MariaDB container is ready"
 log "Setting up database"
-docker exec -i $(docker ps | grep velum-dashboard | awk '{print $1}') rake db:setup
-docker exec -e RAILS_ENV=test -i $(docker ps | grep velum-dashboard | awk '{print $1}') rake db:setup
+until docker exec -i $(docker ps | grep velum-dashboard | awk '{print $1}') rake db:setup;
+do
+    log "Waiting for mariadb to accept connections"
+    sleep 5
+done
+until docker exec -e RAILS_ENV=test -i $(docker ps | grep velum-dashboard | awk '{print $1}') rake db:setup;
```

Comment:
That's why the `until` loops exist - this script can detect the presence of the file before it's actually listening on it. I see @MaximilianMeister's point where the second one shouldn't be necessary.